ccd-definition/CaseField.json

```diff
@@ -187,6 +187,23 @@
     "FieldType": "Applicant",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "applicants",
+    "Label": "Applicant (NEW)",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "partyApplicant",
+    "SecurityClassification": "Public"
+  },
```

Comment:
lets set this to what it should be in production. Remove (NEW)

Reply:
FieldTypeParameter we should name ApplicantParty I think as well, I shall comment again when I get to it

Reply:
👍

Reply:
That is till not actioned.

Reply:
@swalker125 oldApplicant is called Applicant also so we renamed this the new one to Applicants what should i rename to or should i leave as Applicants as it is a collection of Applicants

Reply:
please see https://github.com/hmcts/fpl-ccd-configuration/pull/346#discussion_r299099236

Reply:
It should be Applicant. You have a Collection of Applicant not a collection of applicants... But you're right, that Type has been defined in the old definition stuff. Could you think of another singular type we could use in the mean time? Maybe Applicant1? Then there will need to be a further step to your migration to change Applicant1 -> Applicant. Thoughts?

Reply:
Is this necessary? It seems like a big change if this will not effect the over all result? If it works surely changing an s on the end to a 1 wouldnt really make any difference to the end result? Maybe I am wrong but thoughts.

Reply:
@swalker125